crates/core/src/db/datastore/traits.rs

```diff
@@ -65,16 +65,16 @@ use spacetimedb_table::table::RowRef;
 /// The ANSI SQL standard defined three anomalies in 1992:
 ///
 /// - Dirty Reads: Occur when a transaction reads data written by a concurrent
-/// uncommitted transaction.
+///   uncommitted transaction.
 ///
 /// - Non-repeatable Reads: Occur when a transaction reads the same row twice
-/// and gets different data each time because another transaction has modified
-/// the data in between the reads.
+///   and gets different data each time because another transaction has modified
+///   the data in between the reads.
 ///
 /// - Phantom Reads: Occur when a transaction re-executes a query returning a
-/// set of rows that satisfy a search condition and finds that the set of rows
-/// satisfying the condition has changed due to another recently-committed
-/// transaction.
+///   set of rows that satisfy a search condition and finds that the set of rows
+///   satisfying the condition has changed due to another recently-committed
+///   transaction.
```

Comment:
Formatting change here appears unrelated to stated goal of this PR.

Reply:
A lot of time could have been saved on review if this PR had a description which said that there was a new lint that imposed these changes.